src/components/Contacts/ContactDetails/ContactDontationsTab/PartnershipInfo/PartnershipInfo.tsx

```diff
@@ -50,27 +60,47 @@ const PartnershipTitle = styled(Typography)(({ theme }) => ({
   margin: theme.spacing(1),
 }));
 
+const PartnershipEditIcon = styled(CreateIcon)(({ theme }) => ({
+  width: '18px',
+  height: '18px',
+  margin: theme.spacing(0),
+  color: theme.palette.cruGrayMedium.main,
+}));
+
 interface PartnershipInfoProp {
   contact: ContactDonorAccountsFragment | null;
 }
 
 export const PartnershipInfo: React.FC<PartnershipInfoProp> = ({ contact }) => {
   const { t } = useTranslation();
+  const [editPartnershipModalOpen, setEditPartnershipModalOpen] = useState(
+    false,
+  );
 
   return (
     <PartnershipInfoContainer>
-      <PartnershipTitle variant="h6">{t('Partnership')}</PartnershipTitle>
+      <Box display="flex" justifyContent="space-between">
+        <PartnershipTitle variant="h6">{t('Partnership')}</PartnershipTitle>
+        <IconButton
+          onClick={() => setEditPartnershipModalOpen(true)}
+          aria-label={t('Edit Icon')}
+        >
+          <PartnershipEditIcon />
+        </IconButton>
+      </Box>
       <IconAndTextContainer>
         <IconContainer>
           <HandshakeIcon />
         </IconContainer>
         <Box style={{ margin: 0, padding: 0 }} role="cell">
-          <LabelsAndText variant="subtitle1">{status}</LabelsAndText>
```

Comment:
I'm not sure what ```status``` this is exactly. It definitely wasn't the contact status 🤷‍♂️